src/utils/privileges.cpp

```diff
@@ -262,6 +270,8 @@ void drop_root() {
 	#else
 		_note("This security operation is not available on this system, ignoring");
 	#endif
+	datastore::set_home(home_dir);
+	return home_dir;
```

Comment:
❓ btw why it both applies the home change, and returns the home string to caller?
does any caller use/need this?